.gitattributes

```diff
@@ -0,0 +1,149 @@
+# Auto-detect text files
+* text=auto
```

Comment:
:memo: If the repository requires the use of a specific line ending by all users, it can be added to this line, and we can remove everything below except the *exceptions* to this case. However, the preferable case is to not require a specific line ending and instead allow it to be controlled by the user (or automated system) which checks out the code. For example, a build machine which validates line endings (I believe this is "hygiene" or something), could use the following configuration without being a noisy neighbor to all other contributors:

```
git config core.eol lf
```